content/components/heading.mdx

```diff
@@ -28,12 +28,46 @@ Headings are used to structure the content on the page, with each heading repres
 
 The heading component will render an html h2 tag without any default styling.
 
-## Accessibility
+## Accessibility and usability expectations
+
+A heading is used to provide a description to the section of content that follows it. There are, therefore, a number of expectations for the accessibility and usability of headings:
+
+- The text of the heading is descriptive of the content that follows it.
+- All content associated with the heading follows it in the document source order. While content may be visually styled to appear before the heading, for example a tag that provides categorisation or other content, it should be placed after the heading in the document source order.
+- Overall, heading levels must reflect a logical, page-wide hierarchy, similar to a book table of contents. If the content related to a heading includes further sub-headings, these must hierarchically be at least on level deeper. Ideally, no heading levels should be skipped. For example, an `<h2>` should be followed by an `<h3>`. However, there may be situations where this is not possible (generally, when content is included/embedded dynamically) – in these cases, it is acceptable to skip levels, as long as the sub-headings don't have the same or higher level as their logical "parent" heading.
+
+When using a heading, the level of the heading must be appropriate for its place within the structure of the document, and this should be annotated in designs when possible.
 
 While sighted users rely on visual cues such as font size changes to determine what the heading is, assistive technology users rely on programatic cues that can be read out. When text on a page is visually implied to be a heading, ensure that it is coded as a heading. Additionally, visually implied heading level and coded heading level should be consistent. See [WCAG success criteria: 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html).
 
 Headings allow assistive technology users to quickly navigate around a page. Navigation to text that is not meant to be a heading can be a confusing experience. [Learn more about best heading practices (WAI Headings)](https://www.w3.org/WAI/tutorials/page-structure/headings/).
 
+## Built-in accessibility features
+
+The heading component exposes both the heading role and heading level.
+
+The heading component provides styles with appropriate text sizes to provide a visual hierarchy when the appropriate level is set.
+
+The default heading text has sufficient color contrast against a white background.
```

Comment:
We can probably remove this because this is true of all text, not just heading text. I don't mind keeping it though.